hswindow/hseditor/hsdc6editor/dc6_editor.go

```diff
@@ -75,12 +75,10 @@ func (e *DC6Editor) Build() {
 			e.Path.GetUniqueID()+"selectPalette",
 			e.Project,
 			e.config,
+		).IsOpen(&e.selectPalette).OnSelect(
```

Comment:
I'm not a huge fan of how this was written, but I'll take care of it in another code cleanup PR later.

I'd really like to keep the code from being hard to read :
```golang
if e.selectPaletteWidget == nil {
	id := e.Path.GetUniqueID()+"selectPalette"
	
	onSelectPalette := func(palette *[256]d2interface.Color) {
		e.palette = palette
	}
	
	e.selectPaletteWidget = hswidget.NewSelectPaletteWidget(id, e.Project, e.config).
		IsOpen(&e.selectPalette).
		OnSelect(onSelectPalette)
}
```

Reply:
see a581bd1